EmployeeSalaryApplication/EmployeeSalaryApplication/Program.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace EmployeeSalaryApplication
+{
+    class Program
+    {
+
+        static void printMenu()
+        {
+            Console.WriteLine("1. See all employees.");
+            Console.WriteLine("2. Update the salary of an employee.");
+            Console.WriteLine("3. See salary modifications.");
```

Comment:
you may consider adding an option to print the history salary of a single employee

Reply:
Thank you!